lib/improver/tests/utilities/test_load.py

```diff
@@ -150,6 +155,48 @@ def test_ordering(self):
         self.assertArrayAlmostEqual(result.coord_dims("latitude")[0], 2)
         self.assertArrayAlmostEqual(result.coord_dims("longitude")[0], 3)
 
+    def test_ordering_for_percentile_over_coordinate(self):
+        """Test that the cube has been reordered, if it is originally in an
+        undesirable order and the cube contains a "percentile_over"
+        coordinate."""
+        directory = mkdtemp()
+        filepath = os.path.join(directory, "temp.nc")
+        cube = set_up_percentile_temperature_cube()
+        percentile_points = np.array([10, 50, 90])
```

Comment:
If this array is even needed, it should be cube.coord('percentile_over_realization').points.copy().

Having said this, I see that the same idea is used in the setUp() function, so there is a precedent.

Reply:
Removed.